fcrepo-webapp/src/main/webapp/static/constraints/owl2html.xsl

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Live, I was not even sure how to get it working so I didn't get any further into the guts of how one would do that.